roadmap/implementers-guide/src/node/approval/approval-voting.md

```diff
@@ -0,0 +1,234 @@
+# Approval Voting
+
+## Protocol
+
+Input:
+  - `ApprovalVotingMessage::CheckAndImportAssignment`
+  - `ApprovalVotingMessage::CheckAndImportApproval`
+  - `ApprovalVotingMessage::ApprovedAncestor`
+
+Output:
+  - `ApprovalNetworkingMessage::DistributeAssignment`
+  - `ApprovalNetworkingMessage::DistributeApproval`
+  - `RuntimeApiMessage::Request`
+  - `ChainApiMessage`
+  - `AvailabilityRecoveryMessage::Recover`
+  - `CandidateExecutionMessage::ValidateFromExhaustive`
+
+## Functionality
+
+The approval voting subsystem is responsible for casting votes and determining approval of candidates and as a result, blocks.
```

Comment:
Missing a verb here? `creates blocks.` or `approves blocks`?

Reply:
maybe add a comma like so?
"[...] approval of candidates and, as a result, blocks."